### ISSUE TITLE
Contact page - Broken same-page link

### ISSUE THREAD

**Issue by adritek** [openSUSE/mentoring#221]
### Accessibility
**Page**  
[Contact page](https://www.uyuni-project.org/pages/contact.html)
Type: Warning

**Description**  
There are 8 links to another location within the page is present but does not have a corresponding target.

A link to jump to another position within the page assists users in navigating the web page, but only if the link target exists.
A user might be confused when the browser page does not change. In-page links should generally be presented so it is visually and/or contextually apparent that they are in-page link

**Proposed Solution**  
Ensure that the target for the link exists or remove the same-page link.
- Solution 1: Create a permalink that is not a heading with an appropriate aria-label (thus giving it the target a reference)

**Additional Notes**  
- [Notes additional](https://www.w3.org/TR/WCAG22/#operable)

**Screenshot**  
![Screen Shot 2024-05-19 at 1 47 52 am](https://github.com/openSUSE/mentoring/assets/997506/70d653e6-85df-4ff7-8f04-f2e82748d09d)

**Comment by deneb-alpha** [comment]
moved to https://github.com/uyuni-project/uyuni-project.github.io/issues/122
